WordPress/Classes/ViewRelated/System/WPTabBarController+ShowTab.swift

```diff
@@ -69,22 +69,26 @@ extension WPTabBarController {
 
             WPAppAnalytics.track(.editorCreatedPost, withProperties: [WPAppAnalyticsKeyTapSource: source, WPAppAnalyticsKeyBlogID: blogID, WPAppAnalyticsKeyEditorSource: "stories", WPAppAnalyticsKeyPostType: "post"])
 
-            let controller = StoryEditor.editor(blog: blog, context: ContextManager.shared.mainContext, updated: {_ in }, uploaded: { [weak self] result in
-                switch result {
-                case .success:
-                    ()
-                case .failure(let error):
-                    self?.dismiss(animated: true, completion: nil)
-                    let controller = UIAlertController(title: "Failed to create story", message: "Error: \(error)", preferredStyle: .alert)
-                    let dismiss = UIAlertAction(title: "Dismiss", style: .default) { _ in
-                        controller.dismiss(animated: true, completion: nil)
+            do {
+                let controller = try StoryEditor.editor(blog: blog, context: ContextManager.shared.mainContext, updated: {_ in }, uploaded: { [weak self] result in
+                    switch result {
+                    case .success:
+                        ()
```

Comment:
I know this was an existing expression, but what would you think about changing the `()` to a `break` since that's the more expected syntax for a switch/case skip. (Maybe this predates break being added to Swift 🤔)